Test failure: Missing cat on cmd.exe

```
====================================================== FAILURES =======================================================
__________________________________ TestHDUListFunctions.test_write_hdulist_to_stream __________________________________

self = <astropy.io.fits.tests.test_hdulist.TestHDUListFunctions object at 0x000001E6A475FBA8>

    def test_write_hdulist_to_stream(self):
        """
        Unit test for https://github.com/astropy/astropy/issues/7435
        Ensure that an HDUList can be written to a stream in Python 2
        """
        data = np.array([[1,2,3],[4,5,6]])
        hdu = fits.PrimaryHDU(data)
        hdulist = fits.HDUList([hdu])

        with open(self.temp('test.fits'), 'wb') as fout:
            with subprocess.Popen(["cat"], stdin=subprocess.PIPE,
>                                 stdout=fout) as p:

astropy\io\fits\tests\test_hdulist.py:1054:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
...\lib\subprocess.py:775: in __init__
    restore_signals, start_new_session)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <subprocess.Popen object at 0x000001E6A475FC50>, args = 'cat', executable = None, preexec_fn = None
close_fds = False, pass_fds = (), cwd = None, env = None
startupinfo = <subprocess.STARTUPINFO object at 0x000001E6A475FCC0>, creationflags = 0, shell = False
p2cread = Handle(3672), p2cwrite = 14, c2pread = -1, c2pwrite = Handle(3660), errread = -1, errwrite = Handle(3684)
unused_restore_signals = True, unused_start_new_session = False

    def _execute_child(self, args, executable, preexec_fn, close_fds,
                       pass_fds, cwd, env,
                       startupinfo, creationflags, shell,
                       p2cread, p2cwrite,
                       c2pread, c2pwrite,
                       errread, errwrite,
                       unused_restore_signals, unused_start_new_session):
        """Execute program (MS Windows version)"""

        assert not pass_fds, "pass_fds not supported on Windows."

        if not isinstance(args, str):
            args = list2cmdline(args)

        # Process startup details
        if startupinfo is None:
            startupinfo = STARTUPINFO()
        else:
            # bpo-34044: Copy STARTUPINFO since it is modified above,
            # so the caller can reuse it multiple times.
            startupinfo = startupinfo._copy()

        use_std_handles = -1 not in (p2cread, c2pwrite, errwrite)
        if use_std_handles:
            startupinfo.dwFlags |= _winapi.STARTF_USESTDHANDLES
            startupinfo.hStdInput = p2cread
            startupinfo.hStdOutput = c2pwrite
            startupinfo.hStdError = errwrite

        attribute_list = startupinfo.lpAttributeList
        have_handle_list = bool(attribute_list and
                                "handle_list" in attribute_list and
                                attribute_list["handle_list"])

        # If we were given an handle_list or need to create one
        if have_handle_list or (use_std_handles and close_fds):
            if attribute_list is None:
                attribute_list = startupinfo.lpAttributeList = {}
            handle_list = attribute_list["handle_list"] = \
                list(attribute_list.get("handle_list", []))

            if use_std_handles:
                handle_list += [int(p2cread), int(c2pwrite), int(errwrite)]

            handle_list[:] = self._filter_handle_list(handle_list)

            if handle_list:
                if not close_fds:
                    warnings.warn("startupinfo.lpAttributeList['handle_list'] "
                                  "overriding close_fds", RuntimeWarning)

                # When using the handle_list we always request to inherit
                # handles but the only handles that will be inherited are
                # the ones in the handle_list
                close_fds = False

        if shell:
            startupinfo.dwFlags |= _winapi.STARTF_USESHOWWINDOW
            startupinfo.wShowWindow = _winapi.SW_HIDE
            comspec = os.environ.get("COMSPEC", "cmd.exe")
            args = '{} /c "{}"'.format (comspec, args)

        # Start the process
        try:
            hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                                     # no special security
                                     None, None,
                                     int(not close_fds),
                                     creationflags,
                                     env,
                                     os.fspath(cwd) if cwd is not None else None,
>                                    startupinfo)
E                                    FileNotFoundError: [WinError 2] Das System kann die angegebene Datei nicht finden

...\lib\subprocess.py:1178: FileNotFoundError
============================ 1 failed, 739 passed, 11 skipped, 2 xfailed in 38.08 seconds =============================
```

The error message is probably "Cannot find file". And I believe it's because my Windows machine doesn't have a `cat` executable.

--- a/astropy/io/fits/tests/test_hdulist.py
+++ b/astropy/io/fits/tests/test_hdulist.py
@@ -1040,7 +1040,7 @@ class TestHDUListFunctions(FitsTestCase):
             assert len(hdul) == 5
             assert hdu_popped is hdu1
 
-    # Skip due to ttps://github.com/astropy/astropy/issues/8916
+    # Skip due to https://github.com/astropy/astropy/issues/8916
     @pytest.mark.skipif('sys.platform.startswith("win32")')
     def test_write_hdulist_to_stream(self):
         """

--- a/astropy/io/fits/tests/test_hdulist.py
+++ b/astropy/io/fits/tests/test_hdulist.py
@@ -1040,12 +1040,14 @@ class TestHDUListFunctions(FitsTestCase):
             assert len(hdul) == 5
             assert hdu_popped is hdu1
 
+    # Skip due to ttps://github.com/astropy/astropy/issues/8916
+    @pytest.mark.skipif('sys.platform.startswith("win32")')
     def test_write_hdulist_to_stream(self):
         """
         Unit test for https://github.com/astropy/astropy/issues/7435
-        Ensure that an HDUList can be written to a stream in Python 2
+        to ensure that an HDUList can be written to a stream.
         """
-        data = np.array([[1,2,3],[4,5,6]])
+        data = np.array([[1, 2, 3], [4, 5, 6]])
         hdu = fits.PrimaryHDU(data)
         hdulist = fits.HDUList([hdu])
 

